socorro/siglists/README.rst

```diff
@@ -0,0 +1,195 @@
+change in Socorro via bugs, so it's important that each commit has one
+associated to it. File a bug in the `Socorro::General component
+<https://bugzilla.mozilla.org/enter_bug.cgi?product=Socorro&component=General>`__,
+describe the changes you want to make, and assign it to you.
+
+Then proceed to making those changes...
+
+
+Using the command line
+~~~~~~~~~~~~~~~~~~~~~~
+
+If you are a git power user, you probably don't need us to explain how to do
+this! :)
+
+If you are not, you're probably better off using GitHub's interface. Read on!
+
+
+Using GitHub's interface
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, you need to be logged in to GitHub. Open the file you want to edit, and
+then click the little pen in the top right corner of the page, the one that says
+*Fork this project and edit the file*, or *Edit the file in your fork of this
+project* if you already have a fork of it.
+
+That will take you to an editor, where you can write any changes you want. Once
+you are done editting the file, enter a commit description. We have some
+conventions, and a bot that will automatically close bugs, so please make your
+commit message following this pattern: *Fixes bug XYZ - Desciption of the
+change*. Once you are ready, click *Propose file change*.
+
+That will create a branch in your fork of the socorro project, and take you to
+the commit you just created. You can verify that the changes you made are
+correct, and then click *Create pull request*, and then *Create pull request*
+again. Once the pull request is opened, `Travis CI
+<https://travis-ci.org/mozilla/socorro>`_ will automatically start running our
+test suite, which includes sanity checks for those signature lists. You can see
+the status of those tests in the pull request, and click the *Details* link to
+see logs in case of a failure.
+
+That's it! You have proposed a change, we have been notified about it. Someone
+from the Socorro team will review your changes and merge them if they are
+appropriate. Thank you for contributing to Socorro!
+
+
+Watching only the siglists folder
+---------------------------------
+
+If you are interested in watching what's changing in the ``siglists`` directory
+in the repository, but don't care much about what happens in the rest of the
+Socorro repo, you can easily set a filter in your email client to do that.
+Here's an example filter you can use today:
+
+.. raw:: html
+
+   <pre>to:(socorro@noreply.github.com) ("A socorro/siglists/" OR "M socorro/siglists/" OR "D socorro/siglists")</pre>
+
+
+How to review a siglist change
+------------------------------
+
+The first step is to verify that there is no typo in the change (usually, the
+bug contains examples of crash reports that should be impacted, look at their
+frames). Note that we have a unit test that verifies there are no syntax errors
+in those files.
+
+Run the pull request changes through signature generation using the command line
```

Comment:
This could stand to have a link to the doc section about the command line interface.

Reply:
Oo--good idea!